doc/usage_recomendation.rst

```diff
@@ -0,0 +1,46 @@
+.. _scenario: options.html#scenario
+.. _PCS: options.html#paramcs
+.. _TAE: tae.html
+
+Usage Recommendation
+===========
+*SMAC* of course itself offers a lot of design choices, some of which are crucial to achieve peak performance. Luckily, often it is sufficient to distinguish between a few problem classes.
+To make the usage of *SMAC* as easy as possible, we provide several facades designed for these different use cases. Here we give some general recommendations on
+when to use which facade. This recommendation is based on our experience and technical limitations and is by far not intended to be complete:
+
++-----------------------+----------------------+-----------------------+-------------------------+
+|                       | SMAC4BO              | SMAC4HPO              | SMAC4AC                 |
++=======================+======================+=======================+=========================+
+| # parameter           | low                  | low/medium/high       | low/medium/high         |
++-----------------------+----------------------+-----------------------+-------------------------+
+| categorical parameter | no                   | supported             | supported               |
++-----------------------+----------------------+-----------------------+-------------------------+
+| conditional parameter | no                   | supported             | supported               |
++-----------------------+----------------------+-----------------------+-------------------------+
+| instances             | no                   | None or CV-folds      | yes                     |
++-----------------------+----------------------+-----------------------+-------------------------+
+| stochasticity         | no                   | supported             | supported               |
++-----------------------+----------------------+-----------------------+-------------------------+
+| objective             | any (except runtime) | e.g. validation loss  | e.g. runtime or quality |
++-----------------------+----------------------+-----------------------+-------------------------+
+
+Some examples of typical use cases:
+
+*SMAC4BO*: Vanilla Bayesian Optimization using a *Gaussian Process* and *Expected Improvement*
```

Comment:
I'm wondering if this is vanilla BO as we integrate the hyperparameters of the Gaussian Process.